docs/examples/vector_stores/VearchDemo.ipynb

```diff
@@ -0,0 +1,194 @@
+{
```

Comment:
<p>why the response mixed with Chinese tokens?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/10351/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>